src/sqlancer/cnosdb/ast/CnosDBBinaryArithmeticOperation.java

```diff
@@ -0,0 +1,123 @@
+package sqlancer.cnosdb.ast;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.BinaryOperator;
+
+import sqlancer.Randomly;
+import sqlancer.cnosdb.CnosDBSchema.CnosDBDataType;
+import sqlancer.cnosdb.ast.CnosDBBinaryArithmeticOperation.CnosDBBinaryOperator;
+import sqlancer.common.ast.BinaryOperatorNode;
+
+public class CnosDBBinaryArithmeticOperation extends BinaryOperatorNode<CnosDBExpression, CnosDBBinaryOperator>
+        implements CnosDBExpression {
+
+    public CnosDBBinaryArithmeticOperation(CnosDBExpression left, CnosDBExpression right, CnosDBBinaryOperator op) {
+        super(left, right, op);
+    }
+
+    @Override
+    public CnosDBConstant getExpectedValue() {
```

Comment:
Currently, the PR includes many `getExpectedValue` implementations, but does not support PQS, for which they are needed. Perhaps we can remove these implementations?

Reply:
Thanks, I remove those dead code.